docs/concepts/programming_constructs.md

```diff
@@ -159,10 +159,26 @@ location, then in another program you can import it as a module, using the
     import "<module_path>" as local_name;
 ```
 
-How the `<module_path>` is resolved depends on the specific capabilities of the
-application in which Juttle is running. For the Juttle CLI, it can be either a
-relative or absolute filesystem path or a URL to a remote Juttle file containing
-exported code.
+`<module_path>` can either refer to a *User Import* or a *System
+Import*.
```

Comment:
Do we really want these to be Capitalized Terms? Makes ours docs look a bit like legalese…